src/main/java/com/auth0/json/mgmt/ResourceServer.java

```diff
@@ -28,138 +29,129 @@
     private Boolean isSystem;
 
-    public ResourceServer(String identifier) {
+    @JsonCreator
+    public ResourceServer(@JsonProperty("identifier") String identifier) {
         this.identifier = identifier;
     }
 
-    @JsonCreator
-    public ResourceServer(@JsonProperty("id") String id,
-                          @JsonProperty("name") String name,
-                          @JsonProperty("identifier") String identifier,
-                          @JsonProperty("scopes") List<Scope> scopes,
-                          @JsonProperty("signing_alg") String signingAlg,
-                          @JsonProperty("signing_secret") String signingSecret,
-                          @JsonProperty("allow_offline_access") Boolean allowOfflineAccess,
-                          @JsonProperty("skip_consent_for_verifiable_first_party_clients")
-                                  Boolean skipConsentForVerifiableFirstPartyClients,
-                          @JsonProperty("token_lifetime") Integer tokenLifetime,
-                          @JsonProperty("verificationKey") String verificationKey,
-                          @JsonProperty("verificationLocation") String verificationLocation,
-                          @JsonProperty("is_system") Boolean isSystem) {
-        this.id = id;
-        this.name = name;
-        this.identifier = identifier;
-        this.scopes = scopes;
-        this.signingAlg = signingAlg;
-        this.signingSecret = signingSecret;
-        this.allowOfflineAccess = allowOfflineAccess;
-        this.skipConsentForVerifiableFirstPartyClients = skipConsentForVerifiableFirstPartyClients;
-        this.tokenLifetime = tokenLifetime;
-        this.verificationKey = verificationKey;
-        this.verificationLocation = verificationLocation;
-        this.isSystem = isSystem;
+    public ResourceServer() {
     }
 
+    @JsonProperty("id")
     public String getId() {
         return id;
     }
 
+    @JsonProperty("id")
     public void setId(String id) {
         this.id = id;
     }
 
+    @JsonProperty("name")
     public String getName() {
         return name;
     }
 
+    @JsonProperty("name")
     public void setName(String name) {
         this.name = name;
     }
 
-    public Boolean getIsSystem() {
+    @JsonProperty("is_system")
+    public Boolean isSystem() {
         return isSystem;
     }
 
-    public void setIsSystem(Boolean system) {
-        isSystem = system;
-    }
-
+    @JsonProperty("identifier")
     public String getIdentifier() {
         return identifier;
     }
 
-    public void setIdentifier(String identifier) {
-        this.identifier = identifier;
-    }
-
+    @JsonProperty("scopes")
     public List<Scope> getScopes() {
         return scopes;
     }
 
+    @JsonProperty("scopes")
     public void setScopes(List<Scope> scopes) {
         this.scopes = scopes;
     }
 
-    public String getSigningAlg() {
-        return signingAlg;
+    @JsonProperty("signing_algorithm")
```

Comment:
Remember to also fix this **json property** name.

Reply:
done